### PR TITLE
Tether spring cleaning

### LINKED ISSUE #2570: Few folders around places that need them.
#### Brief description of the issue
Medical only has folders in the Psych office and CMO office. No other folder anywhere. Even in virology. 
Cargo only has one folder with the exception of the QM one. 

#### What you expected to happen
Find proper paperwork gear.

#### What actually happened
Did not.

#### Steps to reproduce
None.
#### Code Revision
Server revision: master - 2017-12-02
8c376610f857d2df29dee93920f7b255ee9724a0 
 
#### Anything else you may wish to add:
- I feel like cargo could have 3-4 folders. Virology 2. The desk of medical one. The main room of medical one too.  Different colors appreciated!

### LINKED ISSUE #4595: Turrets outside AI core are aggressive.
#### Brief description of the issue
While the stairs and the turrets being present up there instead of the ladder in new upload->core transition is nice, turrets outside core where they were not before are not tied to same console as turrets downstairs. They will aggro at you for just going up when you are technically still in same room, just a level higher?

#### What you expected to happen
Turrets outside core itself to not be there or to behave in match with turrets downstairs

#### What actually happened
Almost got shot.

#### Steps to reproduce
Enter upload, see turrets not shoot you. Go upstairs, but not into core itself, see them ready to fire, bolt.

#### Code Revision
fe04192c11e2ad7357512d5a873c44a4ea687d3

### LINKED ISSUE #4679: Tether is missing landmarks for Electrical Issue event
#### Brief description of the issue
Tether has no landmarks named "lightsout", so event never does anything.

#### What you expected to happen
A random area is picked and overloaded, lights break

#### What actually happened
No places to be epicenter exist, nothing happens after announcement

#### Steps to reproduce
- Look around map for "lightsout" landmark
- Shrug in disappointment

#### Code Revision
- 1fe04192c11e2ad7357512d5a873c44a4ea687d3

### LINKED ISSUE #4717: Research and Development area covers unconnected rooms
#### Brief description of the issue
Currently research and development area covers two entirely different nigh-unrelated rooms: third floor of science, the little hub between RD, breakroom, robotics and lobby desk, where stairs are; And research and development lab. Those two areas are not directly connected in any way, outside of labeling and shouldn't be marked as same area for purposes of alarms, power distribution, etc.

#### What you expected to happen
Two different areas for the Research Division main area and Research and Development lab

#### What actually happened
Two unconnected places for some reason are same area.

#### Steps to reproduce
- Look at areas around research.

#### Code Revision
- 1fe04192c11e2ad7357512d5a873c44a4ea687d3 
 
#### Anything else you may wish to add:
The area that separates the two IS in fact its own, Research locker room, so its not like brig, where two z-level-connected parts of same room are same area, but in fact two entirely separate rooms.

### PR DESCRIPTION
**Pull request by Heroman3003**
Overall changes:

- A lot of air alarms and firelock shutters added where they were missing, mostly in maint. There is a lot of cases where its just added firelock/alarm on the joint, so only gonna mention notable one or ones with other changes going forward

- Removed all pipe endcaps in construction areas on asteroid, both because its irrelevant as shown by surface ones, because sprite is broken and because pipes in those places look more authentic cut off, rather than capped

- EDIT2: Added landmarks for electronic overload event, one per z-level. Fixes #4679 

- EDIT2: One new in-wall fluff loot spot

- EDIT3: All turret consoles now follow same logic when choosing area of effect. Just for consistensy, should not affect any existing ones

- EDIT5: Added some folders around medical and cargo. Fixes #2570

- EDIT6: Fixed fulltile polarized windows being broken by one of new PRs

Surface 1:

- Added additional shutter to the security checkpoint for proper insulation in gas leak event, and air alarm to account for new isolated area

- Added missing air alarms in research lower hallway's isolated areas

- Added air alarm to cover both isolated halves of xenoflora atmos

- Removed air alarm in construction area near public EVA due to lack of power as of new update and having players build it

Surface 2:

- Weapons testing range now has air alarms in both halves

- Moved trash pile from maint hydro to nearby maint hallway, makes it better scene cranny without being interrupted by trash divers

- Replaced Black Insulated Gloves with Elastic Black Gloves. Instead of being a rename of combat gloves, a rename of black gloves. New description: "They seem to be made of rubber-like material, though insulation quality is doubtful"

- Added a second air alarm to second floor atrium because screw area that big, even if single atmos zone, having only one air alarm

- In similar fashion, removed one of two air alarms in maintenance near BDSM dungeon because they were on adjacent tiles and way too redundant

Surface 3:

- Replaced Research and Development in the big area between RD, breakroom and Robotics with new area, Research Division. Added APC for it. Fixes #4717

- Added firelocks between Mech Bay and Robotics

- Added lattice to open space in Pump Station

- Replaced some regular firelocks near pool with glass ones for consistency and practicality

- Removed redundant air alarm and firelock line in gateway hallway

- Moved firelock line near southern bar entrance slightly to allow for insertion of air alarm

- Moved trashpile out of Locker Room construction area and into maint before it. Replaced engineering-only airlock with public access one. Same intent as with maint hydro, should make it a better place to lewd or furnish

- EDIT1: Added air alarm to the kitchen

Asteroid 1

- Added vent and scrubber to the backup atmos

- Removed lines of windows in exploration and on bridge for consistensy and visual appeal. Though kept ones in HoP's office because idea of people being angry enough for them to be 'outside-reinforced' is funny

- Added firelocks to windows and doors in the edge of construction area

- Added an unfinished disposal pipe leading to construction area, similar to one second floor of it has

- Additional vent and scrubber across from tool storage

- CE window tint now covers external windows too

- EDIT2: Removed random meter on the floor in front of shuttle. It was doing absolutely nothing, possibly unremoved leftover?

- EDIT2: Adjusted Pathfinder's Office area bounds slightly

Asteroid 2

- Moved line of shutter near staircase to better isolate ventable areas

- Added air alarms in the staircase

- Added shutters to external windows in construction area

- Construction area's area slightly adjusted to allow airlock to maint to function despite area being unpowered now

- Fixed weird chapel morgue fire extinguisher being a tile off and not on wall, but on floor

- EDIT3: In upload console room, there is now a control panel for turrets right upstairs. Fixes #4595 

Asteroid 3

- Fixed disconnected vent in blue armory

- Added new line of firelocks and air alarm near cargo lobby

- Properly isolated hallway area on psychiatrist corner, now complete with air alarm and scrubber

- Moved one of vents in brig hallway to not be under shutter

- Added a bit of rwall and appropriate area to them behind firing range to complete the line because otherwise I'll never look at that place same again

- Added air alarm to a second half of firing range

- Improved tinting of windows in HoS, CMO, IAA offices, patient rooms, exam room and recovery ward, now both full and one-directional parts of windows will tint

- Added a missed APC to the security maintenance, locker that was there moved one screen south

- EDIT4: Added a false wall access to construction area near AI core.

tl;dr: fixing a shiton of small mapping issues, primarily related to atmos and maints, in addition to some oversights needing adressing from last update, weird zonings and placement bugs, improving some things that require just a touch, and all of that throughout entire Tether.

### CODE REVIEW

**Comment by TheFurryFeline** [comment]
Think kitchen air alarm would be nice to include, separate from the whole bar?

**Comment by Heroman3003** [comment]
I knew I couldn't have gotten it all. Thanks, added.

**Comment by Heroman3003** [comment]
Okay, Travis seems mad at active edge now, but its on z-18 which I did not touch, so due to same bug as before, it seems.
